### ISSUE TITLE
After executing fix_host_save.cmd, there is no change in the terminal

### ISSUE THREAD

**Issue by Flikify** [deafdudecomputers/PalWorldSaveTools#21]
![image](https://github.com/user-attachments/assets/0f74ce02-83ab-4b0f-9d8d-4026fb30cc6a)
After executing fix_host_save.cmd, there is no change in the terminal. I tried several times and changed to other computers, but it is still the same.
Is there any solution? Thank you

**Comment by deafdudecomputers** [comment]
Is your internet on? Did you download AIO package as mentioned in readme? Did you download the python directly from official source, python.org/download? Did you confirm that both options are ticked when you install the python? And can you confirmthe external lib folder of my tool is being populated? Please answer these questions so I can see what's going on. Thank you.

**Comment by Flikify** [comment]
Thank you very much for your reply.
I did everything you said, but the problem still cannot be solved.
I tried to check the code, and I debugged and found that there was an error related to subprocess. I commented out some command lines using subprocess, manually installed the required packages to the default site-packages, and after lines 12 and 13 of import_libs.py, the software can run normally.
![image](https://github.com/user-attachments/assets/cd411bea-b3a7-4e0c-afbb-bce382b43d12)
